src/tidecli/utils/file_handler.py

```diff
@@ -273,44 +374,50 @@ def get_task_file_data(file_path: Path, metadata: TaskData) -> list[TaskFile]:
                     metadata_bycode, metadata_gapcode = split_file_contents(
                         f1.content
                     )
-
+    
```

Comment:
Trailing whitespace could be fixed by running the black formatter

Reply:
removed